measures/BuildExistingModel/measure.rb

```diff
@@ -109,6 +110,17 @@ def arguments(model)
     arg.setDescription('If true, output the annual component loads.')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('run_hescore_workflow', false)
+    arg.setDisplayName('Run Home Energy Score workflow')
+    arg.setDescription('If true, applies the HEScoreHPXML measure, as well as the HEScoreRuleset and HPXMLtoOpenstudio measures located in the os_hescore_directory path')
+    arg.setDefaultValue(false)
+    args << arg
+
+    arg = OpenStudio::Ruleset::OSArgument.makeStringArgument('os_hescore_directory', false)
+    arg.setDisplayName('Path to the OpenStudio-HEScore directory')
```

Comment:
And `HEScore Workflow: OpenStudio-HEScore Path`.

Reply:
Better yet, couldn't `os_hescore_directory` being populated imply HEScore Workflow? Meaning, you only need one argument and not two.